packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1164,6 +1165,16 @@ async function handler(
         await usersRepository.updateLastActiveAt(booking.userId);
       }
 
+         if (routingFormResponseId && teamId) {
+      await AssignmentReasonRecorder.routingFormRoute({
```

Comment:
Hopefully by having a class that handles just recording the assignment, it'll make this feature not as coupled to the places we need this in.